transforms/language/html2parquet/dpk_html2parquet/transform.py

```diff
@@ -236,3 +236,35 @@ def apply_input_params(self, args: Namespace) -> bool:
         self.params = self.params | captured
         logger.info(f"html2parquet parameters are : {self.params}")
         return True
+
+
+class Html2Parquet(Html2ParquetTransform):
```

Comment:
This class needs some documentation here and in a README.

Reply:
BIG TIME :-). working on it. Thanks